Bootstrapping fails to create `project-dhall` folder

A follow up for #27. It fails to create the `project-dhall` folder.

```
make -f project-files.mk
rm -rf updo
curl -sSL https://github.com/cabalism/updo/archive/cdb06ebebaf6b8739e900780bc317cec3cac2d24.tar.gz | tar -xz
mv updo-* updo
chmod +x $(grep -RIl '^#!' updo)
...
mkdir -p .updo/ghc-9.4.8/
[ -e ./project-dhall/ghc-9.4.8/deps-external.dhall ] || cp ./updo/empty/source-pkgs.dhall .updo/ghc-9.4.8/deps-external.dhall
mkdir -p .updo/ghc-9.4.8/
[ -e ./project-dhall/ghc-9.4.8/deps-internal.dhall ] || cp ./updo/empty/source-pkgs.dhall .updo/ghc-9.4.8/deps-internal.dhall
mkdir -p .updo/ghc-9.4.8/
[ -e ./project-dhall/ghc-9.4.8/forks-external.dhall ] || cp ./updo/empty/source-pkgs.dhall .updo/ghc-9.4.8/forks-external.dhall
mkdir -p .updo/ghc-9.4.8/
[ -e ./project-dhall/ghc-9.4.8/forks-internal.dhall ] || cp ./updo/empty/source-pkgs.dhall .updo/ghc-9.4.8/forks-internal.dhall
[ -e project-dhall/pkg-groups.dhall ] || cp updo/project-skeleton/pkg-groups.dhall project-dhall/pkg-groups.dhall
cp: cannot create regular file 'project-dhall/pkg-groups.dhall': No such file or directory
make: *** [updo/project-skeleton/Makefile:9: project-dhall/pkg-groups.dhall] Error 1
```

--- a/project-skeleton/Makefile
+++ b/project-skeleton/Makefile
@@ -6,6 +6,7 @@ project-dhall/pkgs/%.dhall: \
 project-dhall/pkg-groups.dhall: \
   updo/project-skeleton/pkg-groups.dhall \
   $(wildcard project-dhall/pkgs/*.dhall)
+	mkdir -p project-dhall
 	[ -e $@ ] || cp $< $@
 
 project-dhall/pkgs-upgrade-todo.dhall: \

